torch/_inductor/codegen/triton.py

```diff
@@ -1887,6 +1887,72 @@ def sum_fn(a, b):
 
         return tuple(result_vars)
 
+    def sort(
+        self,
+        dtypes: Tuple[torch.dtype, ...],
+        values: Tuple[CSEVariable, ...],
+        stable: bool,
+        descending: bool,
+    ) -> Tuple[CSEVariable, ...]:
+        assert self.inside_reduction
+        masks = {f"{tree.prefix}mask" for tree in self.range_trees}
+        self.filter_masks(masks)
+        masks = sorted(masks)
+        assert not self._load_mask, "ops.sort not supported inside ops.masked"
+        assert (
+            self.persistent_reduction
+        ), "ops.sort is only supported in persistent reductions"
+        reduction_range_prefix = self.range_trees[-1].prefix
+
+        broadcasted_values = []
+
+        cse_compute = functools.partial(self.cse.generate, self.compute)
+        dim = self.triton_tensor_ndim() - 1
+
+        for value, dtype in zip(values, dtypes):
+            acc_type = triton_acc_type(dtype)
+            cond = " & ".join(masks)
+
+            value = self.cse.generate(
+                self.compute,
+                f"tl.broadcast_to({value}, {self.dense_size_str()})",
+            )
+            broadcasted_values.append(value)
+
+        def csv(values):
+            return " ".join(f"{value}," for value in values)
```

Comment:
Nit. You can also do
```suggestion
            return ", ".join(str(value) for value in values)
```
and it may be even clearer what you want to do here and you skip the extra comma

Reply:
Yeah that won't be a tuple in the single element case though, not that that is actually used here yet but down the line it should be.

Reply:
This is taken from scan btw where we do need to handle 1 or many